cypress/support/readPdf.ts

```diff
@@ -0,0 +1,15 @@
+import fs from "fs";
+import path from "path";
+import pdf from "pdf-parse";
+
+const readPdf = (pathToPdf: string): Promise<string> => {
```

Comment:
can this be an async function instead?

Reply:
Done! Agreed that the async function is a lot more readable.